app/assets/stylesheets/globals.scss

```diff
@@ -41,7 +41,7 @@ $min-desktop-width: $max-mobile-width + 1px;
   }
   &:active,
   &.active {
-    background-color: variables.$secondary-color;
+    background-color: variables.$secondary-color-dark;
```

Comment:
This makes the active sort link a darker color which improves the contrast with the text.